src/Auditor.php

```diff
@@ -79,10 +81,49 @@ public function execute(AuditableContract $model)
             return;
         }
 
-        if ($audit = $driver->audit($model)) {
+        $uuid = null;
+        $list_of_properties = [];
+        foreach (config('audit.relation_hierarchy', []) as $class_type_to_audit => $local_list_of_methods) {
+            foreach ($local_list_of_methods as $local_method) {
+                if ($model instanceof $class_type_to_audit) {
+                    $list_of_properties[] = $local_method['property'];
+                    $uuid = (string) Uuid::generate();
+                } elseif ($model instanceof $class_type_to_audit) {
```

Comment:
You're doing a `if()` statement followed by an `elseif()` with the exact same thing!

Reply:
My bad. Thanks for the catch.